azurerm/resource_arm_api_management_api.go

```diff
@@ -170,12 +170,12 @@ func resourceArmApiManagementApi() *schema.Resource {
 
 			"version": {
 				Type:     schema.TypeString,
-				Computed: true,
+				Optional: true,
```

Comment:
I believe this'll need to be Optional & Computed, which means that it's configurable (Optional) but can have a default value (Computed) since Azure auto-assigns a version if one is unspecified?

Reply:
When I left `computed: true` during testing, removing the `version` and `version_set_id` from a resource resulted in no change on an apply. But I would expect an API to be able to be removed from a version set. Any ideas why I would see this?

Reply:
My local state also doesn't appear to persist the `version` and `version_set_id`  between applies when `computed: true`.

Reply:
Azure doesn't require or set a version on the first API in a version set. A version is only required on subsequent versions, and that value also is not generated.